DeepSkyStacker/StackedBitmap.cpp

```diff
@@ -547,7 +547,7 @@ HBITMAP CStackedBitmap::GetBitmap(C32BitsBitmap & Bitmap, RECT * pRect)
 		};
 
 #if defined (_OPENMP)
-#pragma omp parallel for default(none)
+#pragma omp parallel for default(none) shared(lYMin)
```

Comment:
I'm quite confused as to why the compiler requires that we specify the data-sharing attribute of `lYMin`, but not that of `lYMax`. Nonetheless this change was enough to satisfy it and compile. Same at line 663.